component/main.jsonnet

```
@@ -6,6 +6,7 @@ local inv = kap.inventory();
 local params = inv.parameters.prometheus_pushgateway;
 
 {
+  [if params.createNamespace then '00_namespace']: kube.Namespace(params.namespace),
```

Use the following to allow Prometheus to pick up the rules in the namespace:

```suggestion
local nsLabels = 
  if inv.parameters.facts.distribution == 'openshift4' then
    {
      'openshift.io/cluster-monitoring': 'true',
    }
  else
    {
      SYNMonitoring: 'main',
    };

{
  [if params.createNamespace then '00_namespace']:
    kube.Namespace(params.namespace) {
      labels+: nsLabels,
    },
```